test/mock/makeMockUseContractAddress.ts

```diff
@@ -0,0 +1,57 @@
+import dotenv from 'dotenv'
+import { Address } from 'viem'
+
+dotenv.config({ path: '.env.local' })
+
+const deploymentsString = process.env.NEXT_PUBLIC_DEPLOYMENT_ADDRESSES || '{}'
+const deploymentAddresses = JSON.parse(deploymentsString)
+const contracts = {
+  ensRegistry: {
+    address: deploymentAddresses.ENSRegistry,
+  },
+  ensUniversalResolver: {
+    address: deploymentAddresses.UniversalResolver,
+  },
+  multicall3: {
+    address: deploymentAddresses.Multicall,
+  },
+  ensBaseRegistrarImplementation: {
+    address: deploymentAddresses.BaseRegistrarImplementation,
+  },
+  ensDnsRegistrar: {
+    address: deploymentAddresses.LegacyDNSRegistrar,
+  },
+  ensEthRegistrarController: {
+    address: deploymentAddresses.ETHRegistrarController,
+  },
+  ensLegacyEthRegistrarController: {
+    address: deploymentAddresses.LegacyETHRegistrarController,
+  },
+  ensNameWrapper: {
+    address: deploymentAddresses.NameWrapper,
+  },
+  ensPublicResolver: {
+    address: deploymentAddresses.PublicResolver,
+  },
+  ensReverseRegistrar: {
+    address: deploymentAddresses.ReverseRegistrar,
+  },
+  ensBulkRenewal: {
+    address: deploymentAddresses.StaticBulkRenewal,
+  },
+  ensDnssecImpl: {
+    address: deploymentAddresses.LegacyDNSSECImpl,
+  },
```

Comment:
not sure if this is actually used but this shouldn't use the legacy deployment address

Reply:
Originally copied the chain mapping from addTestContracts...  I refactored it so that the base data is coming from constants/chains.ts.